app/policies/collection_policy.rb

```diff
@@ -1,13 +1,26 @@
 # frozen_string_literal: true
 
 class CollectionPolicy < ApplicationPolicy
-  pre_check :collection_creator?
+  alias_rule :new?, :create?, :show?, :update?, :edit?, :wait?, to: :manage?
+
+  def manage?
+    record.user_id == user.id
+  end
+
+  def new?
+    collection_creator?
+  end
+
+  def create?
+    collection_creator?
+  end
```

Comment:
@justinlittman I know this was merged already, but I have a question for my own ActionPolicy learning: is it customary to have a policy expressed as both an alias and in its own method? It looks like both `new?` and `create?` are in both spots here. Naïvely, I might expect the explicit `def` methods to take precedence over whatever `alias_rule` does but maybe AP is smarter than the average bear.